lib/Markdent/Role/HTMLStream.pm

```diff
@@ -382,7 +382,8 @@ sub end_code {
         delete @p{ grep { !defined $p{$_} } keys %p };
 
         $self->_stream_start_tag(
-            'a', {
+            'a',
+            {
```

Comment:
I'm not sure why the formatting changed here? Did you do this on purpose. What's really confusing is that I'd expect the tidyall tests to complain but they didn't.

But regardless, can you redo this without changes to the code layout?

Reply:
@autarch : the tidyall tests failed before I ran "tidyall lib..." on the file and passed after I ran it. Perhaps it is the new version of perltidy - see https://www.martinfowler.com/bliki/TwoHardThings.html ("cache"s).

Reply:
Hmm, that's confusing. I have the latest perltidy locally and when I run tidyall locally on the master branch it doesn't change anything.

I think _both_ the original and your changes are accepted by perltidy, basically. That said, I'd prefer to merge the most minimal change rather than mix in a bunch of unrelated formatting changes.